packages/core/src/telemetry/service.test.ts

```diff
@@ -1,23 +1,122 @@
-import { afterEach, beforeEach, expect, test } from "vitest";
+import * as fs from "fs";
+import os from "os";
+import * as process from "process";
+import { afterAll, afterEach, beforeAll, expect, test, vi } from "vitest";
+import createFetchMock from "vitest-fetch-mock";
 
 import { buildOptions } from "@/config/options";
 import { TelemetryService } from "@/telemetry/service";
 
-let originalEnv: NodeJS.ProcessEnv;
+const cpus = os.cpus() || [];
 
-beforeEach(() => {
-  originalEnv = { ...process.env };
+const options = buildOptions({
+  cliOptions: { configFile: "", rootDir: "" },
+});
```

Comment:
If you take a look at `_test/setup.ts`, you can see that we're already doing this and including in the vitest test context that gets passed to every test. So, you should be able to do:
```ts
test("...", async ({ options }) => {
  const service = new TelemetryService({ options })
}
```